src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/LoggerMessageGeneratorParserTests.cs

```diff
@@ -679,6 +680,56 @@ static partial class C
             Assert.Empty(diagnostics);    // should fail quietly on broken code
         }
 
+        //[Fact]
+        //[ActiveIssue("todo", TestPlatforms.Browser)]
```

Comment:
Should these be commented out?

Reply:
This is temporary; trying to understand WASM CI failure.